lexicons/app/bsky/embed/post.json

```diff
@@ -0,0 +1,38 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.embed.post",
```

Comment:
I almost wonder if we should make this `embed.record` since that's what it's actually "describing" & may be more mappable to other data types 🤔 

For instance embedding something like a "profile".

Or (contrived example I use often), but imagine a Wordle app deployed on atproto & wanted to embed your score for the day in a native way?

The possible downside would be overloading this embed type & maybe each of those embeds should be treated differently. But at the same time, I don't see anything in the schema that makes this specific to posts

Reply:
Also should we have a list of "relevant blobs" in `presentedPost`?

Reply:
Very much hear that, good thought. @pfrazee and I worked through some of the tradeoffs here in person.  It's really tempting to do what you're saying, particularly because what we're sending down for a post (uri, cid, author, record) _would_ work equally well for other record types.  The rub is that to do this by the book it opens-up questions about how to model fallback content for clients that don't understand the embed (since simply omitting the unfamiliar embed could violate user intent), and we weren't quite ready to bite that off.  I'm still open to it but that was our thinking a couple days ago.

Reply:
Hmmm I totally see that...

So basically the implication here is that "only embeds of type `app.bsky.feed.post` are allowed", right?

We need to tangle with this fallback question in general soon imo...

Totally understand the concern, but I think I'm in favor of `embed.record` with fallback to "could not display embedded record" element or something

Reply:
But also if you guys already grappled with it, I'm down to trust your judgement & just get this rolling. We can always add in the generic form later

Reply:
There's also a decent argument that there's a difference between:
"embedding this post" (like a quote tweet)
"embedding this record" (like a link w preview)
🤷‍♂️

Reply:
We were trying to avoid having to do schema negotiation tooling now, but it's occurring to me that targeting a specific schema makes it hard for us to v2 the post. Maybe we go generic but don't worry about being super flexible or handling fallbacks well yet?

Reply:
yup I think that's reasonable 👍 

it's in line with the rest of lex: likes can be on _anything_. but if the like doesn't make sense, it's ignored

altho nothing in this prevents a generic record, it's just the implication of "embedding a post" v "embedding a record" & if that's a useful distinction.

In my mind that distinction - if it exists - is: quote tweet v link to tweet w preview